Patch by Mike A. Harris <mharris@redhat.com> to enable enables the "vmware"
driver on AMD64 architecture, as per request of VMware Inc.  This is already
done in CVS HEAD (post 6.8.x)

Resolves request:
	https://bugzilla.redhat.com/beta/show_bug.cgi?id=145588


--- xc/config/cf/xorg.cf.AMD64-override-default-driverlist	2005-01-23 22:11:42.000000000 -0500
+++ xc/config/cf/xorg.cf	2005-01-27 14:26:43.000000000 -0500
@@ -444,7 +444,7 @@
 #  define XF86CardDrivers	mga glint nv tga s3 s3virge sis rendition \
 				neomagic tdfx savage cirrus tseng trident \
 				chips apm i128 ati ark cyrix siliconmotion \
-				vga dummy vesa i810 \
+				vga dummy vesa i810 vmware \
 				XF86OSCardDrivers XF86ExtraCardDrivers
 # endif
 
